CHANGELOG.md

```diff
@@ -7,9 +7,14 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+<<<<<<< HEAD
```

Comment:
We have some rebase artefacts that we'd need to remove before merging.

Reply:
How did that get in there? 🙃  Will sort out.